test/extensione2e/kafka/source_kafka_broker_ksvc_test.go

```diff
@@ -57,18 +57,18 @@ kind: %q`, channelAPIVersion, kafkaChannelKind),
 			Namespace: testNamespace,
 		},
 		Spec: eventingv1.TriggerSpec{
-			Broker: kafkaBrokerName,
+			Broker: kafkaChannelBrokerName,
 			Subscriber: duckv1.Destination{
 				Ref: &duckv1.KReference{
 					APIVersion: ksvcAPIVersion,
 					Kind:       ksvcKind,
-					Name:       helloWorldService + "-broker",
+					Name:       helloWorldService + "-kafkaChannelBroker",
```

Comment:
The uppercase letters are breaking in the test.

Reply:
ah, one more forgotten - saw that already